JapaneseFilter/sections/whitelist.txt

```diff
@@ -3,6 +3,8 @@
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/66610
 @@||identity.mparticle.com/*/login$xmlhttprequest,domain=dplay.jp
+! https://github.com/AdguardTeam/AdguardFilters/issues/69810
+@@||*.jp/banner$domain=milcow.com
```

Comment:
`@@||myfigure.jp/banner/$image,domain=milcow.com` or change it to more specifc.
@Yuki2718 should we unblock it on the website?

Reply:
Probably yes, kinda promoted items but not obtrusive ads. But this site is a portal to show and compare various shopping sites so `@@||*.jp/banner$domain=milcow.com` makes sense, the contents will switch as a matter of time. OTOH you can block `milcow.com##.adaida`.

Reply:
Yeah, .adaida is obv ad leftover, removed.

Reply:
Ok, but I don't see any other requests, except the `myfigure.jp`.
@stanislav-atr please also change the exception.